valuable-derive/Cargo.toml

```diff
@@ -1,10 +1,11 @@
 [package]
 name = "valuable-derive"
 version = "0.1.0"
-authors = ["Carl Lerche <me@carllerche.com>", "Taiki Endo <te316e89@gmail.com>"]
+authors = ["Tokio Contributors <team@tokio.rs>"]
```

Comment:
This field is soft-deprecated, cargo no longer generates this field by default, and Rust's official services no longer reference this field at all. https://rust-lang.github.io/rfcs/3052-optional-authors-field.html

So, we can also remove them.

```suggestion
```